fvm/src/machine/default.rs

```diff
@@ -64,10 +64,15 @@ where
             "initializing a new machine, epoch={}, base_fee={}, nv={:?}, root={}",
             epoch, &base_fee, network_version, state_root
         );
+
+        if network_version != NetworkVersion::V14 {
+            return Err(anyhow!("unsupported network version: {}", network_version));
+        }
```

Comment:
Better change to a `<`, since we'll soon support V15? (and the FVM is really designed to support V14 and above).

Reply:
I'd rather _explicitly_ decide with network versions we support. For example, to support nv15 we'll need support for multiple actor versions.

Reply:
I concur with @Stebalien here